lib/optimizer/tests/src/ErrorComparison.php

```diff
@@ -17,23 +17,24 @@ trait ErrorComparison
     /**
      * Assert that two sets of errors are the same.
      *
-     * @param ErrorCollection|Error[] $expectedErrors Set of expected errors.
-     * @param ErrorCollection|Error[] $actualErrors   Set of actual errors.
+     * @param ErrorCollection|Error[] $expected_errors Set of expected errors.
+     * @param ErrorCollection|Error[] $actualErrors    Set of actual errors.
      */
-    protected function assertSameErrors($expectedErrors, $actualErrors)
+    protected function assertSameErrors($expected_errors, $actualErrors)
```

Comment:
Any reason to change `$expectedErrors` from camel case to snake case?

Reply:
Will revert this one...